### ISSUE TITLE
DSN: Propose editorial interface design to refactor COVID and operating status

### ISSUE THREAD

**Issue by dsinla** [department-of-veterans-affairs/va.gov-cms#10842]
## Description
The current (Sep 2022) bulleted descriptive text for COVID status policies does not meet the operational reality at the facilities - it's not granular enough to give Veterans the variation of policies that are in place at each facility.
VAMC facilities will keep the COVID status of High, Medium, and Low, but we will remove the bulleted text. Editors will use the Operating Status open text field to provide COVID policies specific to their facility.

We will modify the editorial interface to remove the read-only Guidelines area, and to provide better text descriptions for how to complete each field.

Refer to initial decisions here: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10708

![image](https://user-images.githubusercontent.com/62269210/192011030-cdef9a20-6524-4b6e-9455-23976b19f9b3.png)

## Acceptance Criteria
- [ ] A design is proposed - that includes the following:
  - [ ] Editorial interface is reworked to allow an editor to easily craft their COVID-specific policies, and allow for local customization.
  - [ ] Editors have guidance on how to use with various emergencies - inclusive of COVID
  - [ ] Need links to the guidance / example text 
  - [ ] Build in ability to measure success? What metrics? <- Eng ticket (Ie - generate a report of the free text area?)
  - [ ] Interface reinforces the fact that - for the time being with COVID - "Normal service and hours" can not be selected 
    - [ ] How/when to account for the trigger to turn this off?  

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [X] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [X] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by davidmpickett** [comment]
@EWashb this was flagged in Facilities refinement as something that might make a good ticket for Blake to ramp on to next sprint

**Comment by EWashb** [comment]
@davidmpickett I'm unsure if Blake will have capacity next week since he will be working on his onboarding ticket and paperwork. I'll check with @dsinla on the level of priority. I would like to ensure Blake has the appropriate time to complete his onboarding before doing some ticketed work. It will also take him a bit of time to get into the CMS Design System

**Comment by dsinla** [comment]
@davidmpickett  @EWashb Do we expect Blake to be available for Sprint 69? Seems like it? As a good first activity.

**Comment by davidmpickett** [comment]
Adding blocked until Blake is up to speed on Drupal and Figma

**Comment by dsinla** [comment]
Hey @BlakeOrgan assigning to you. If you have any questions about the notes in the graphic above, please reach out.

**Comment by EWashb** [comment]
@dsinla just to confirm, the "normal service hours" will be the only option for operating status? Therefore, we most likely don't need a multi-select?

**Comment by dsinla** [comment]
@EWashb It's the other way around.  For the time being, while VAMCs are still issuing High Medium and Low COVID statuses, an editor should not be able to choose "Normal services and hours".  We're still not back to normal. 

COVID status must be communicated, at a minimum. If there is a hurricane or fire or other more-pressing emergency, than that can bump COVID status. Dave C talked about _"only conveying one emergency at a time"_.  

In other words, no facility should have "Normal services and hours" displayed at this time  - not until we retire COVID status in the future or some other decision is made.

**Comment by davidmpickett** [comment]
I have questions about disabling "Normal Services and Hours". That doesn't align with my understanding of the direction. Maybe the four of us can meet to chat through this tomorrow? @dsinla @EWashb @BlakeOrgan

**Comment by dsinla** [comment]
@davidmpickett I'm happy to chat a bit. I may have been a bit loose with language - we don't have to literally disable the radio button. The goal is to guide and inform editors that for now (while VAMCs are still issuing High, Medium, Low COVID status), all editors are expected to publish that status for their site, and craft the appropriate policy messaging. How best to convey that guidance to editors is one of the key goals of this ticket.  I'd still like to hear you understanding.

**Comment by davidmpickett** [comment]
Did some spelunking. Additional notes/questions for discussion

1. Here is a related open ticket with Sitewide Content with vital context: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45774
2. Has it been determined where the "How to update your VA health facility COVID-19 status" content will live? I know both KB and Sharepoint were raised as possibilities.
3. Is there a FE dev ticket to go along with the CMS dev ticket you linked? I'm unclear on what the FE changes will be. This is also an open question in the Sitewide Content Ticket. 
4. Operating Status "fieldset" is also used on non-VAMC facilities. Restricting the radio button would likely have impacts on every Facility type. But I think the helper text can be unique?

**Comment by dsinla** [comment]
Thanks for meeting - @davidmpickett brought up some good questions that will have to be verified, but in discussions with @EWashb we think @BlakeOrgan can move forward on initial wireframes. Blake- happy to talk again for any info/context you need. I'm sure @JayDarnell will be available for any tech questions as the wireframe is developed, or for a bit of a 'tech review' of the first draft.   

We all agree that CM component here is non-trivial. I will reach out to Blake Scates in VHADM to give him the context, and discuss the next CM steps for educating editors.

**Comment by EWashb** [comment]
Thanks for checking on that @davidmpickett!

**Comment by JayDarnell** [comment]
@davidmpickett the lift for adding custom validation that is the same for all facility types vs one way for VAMCs and another way for other facilities is nearly the same. We would just need to add a couple extra lines of code to grab and check the facility type before applying the validation logic.

**Comment by dsinla** [comment]
@BlakeOrgan My understanding was that @EWashb was going to walk you through some of the fundamentals here for CMS design.  We know she is unfortunately out. Do you expect to see any progress on this issue in this sprint?

**Comment by BlakeOrgan** [comment]
@dsinla No, not for this sprint. @davidmpickett and I chatted about this yesterday and he was able to give context about what some of the expectations and limitations are. I can start to put some preliminary wireframes together starting next sprint, but we will need to make sure everyone is on the same page about what might change within the CMS and how any changes made can/should be reflected on the front end.

I think it would be useful to get me, you, @EWashb, and Dave P together once me and Erika have got some initial ideas put together and see if it aligns with what you and Dave C are expecting.

**Comment by EWashb** [comment]
@BlakeOrgan is working on a mockup to review with Dave C. on 10/27. We will have a technical review before meeting with the PO

**Comment by BlakeOrgan** [comment]
@dsinla

@jcklpe created ticket #11361 to decide how we want the public facing side of this to look. Once we have a clear direction on how that is changing and everyone is on the same page about it, I can provide an editor experience to reflect that.

Until then, [here is a Figma file](https://www.figma.com/file/IYpeP2ZAmBTT8jqhdtJklu/Covid-%2F-Facility-Status?node-id=15%3A33336) with my preliminary designs on some ways that the editor experience could look, but these could definitely change depending on the direction the front end takes.

**Comment by davidmpickett** [comment]
@dawnpruitt @dsinla  Does this meet the ACs for this sprint? 
I would suggest that any follow up editor experience design @BlakeOrgan does after #11361 is complete be a separate follow up ticket.

**Comment by EWashb** [comment]
I'm going to go ahead and close this. the proposal of the interface is complete
